app/Census/CensusOfCanada1931.php

```diff
@@ -0,0 +1,85 @@
+?php
```

Comment:
Missing the opening `<` character

Reply:
Sorry about that, I didn't notice it was missing when I copied and past it into github.  I've added it in now.